build.sbt

```diff
@@ -420,13 +421,15 @@ lazy val compiler = configureAsSubproject(project)
     scalacOptions in Compile in doc ++= Seq(
       "-doc-root-content", (sourceDirectory in Compile).value + "/rootdoc.txt"
     ),
-    Osgi.headers +=
+    Osgi.headers ++= Seq(
       "Import-Package" -> ("jline.*;resolution:=optional," +
                            "org.apache.tools.ant.*;resolution:=optional," +
                            "scala.util.parsing.*;version=\"${range;[====,====];"+versionNumber("scala-parser-combinators")+"}\";resolution:=optional," +
                            "scala.xml.*;version=\"${range;[====,====];"+versionNumber("scala-xml")+"}\";resolution:=optional," +
                            "scala.*;version=\"${range;[==,=+);${ver}}\"," +
                            "*"),
+      "Class-Path" -> "scala-reflect.jar scala-library.jar"
```

Comment:
Ah, I was looking at the wrong manifest. According to http://docs.oracle.com/javase/8/docs/technotes/guides/jar/jar.html#classpath this looks rather application-specific. Does it make sense to include it in a library that gets published to a Maven or Ivy repo? A `Class-Path` would require a specific directory layout locally. The command line use case doesn't look very compelling to me (why use `jrunscript` instead of `scala` from the command line if you have a scala distribution installed?)